portfolio/src/main/webapp/greeter.js

```diff
@@ -0,0 +1,8 @@
+const greeting = async () => {
+    const serverResponse = await fetch('/hello');
+    const text = await serverResponse.text;
+
+
+  const helloContainer = document.getElementById('hello-div');
+  dateContainer.innerText = textFromResponse;
```

Comment:
Please correct the indentation.

Reply:
Kk will do. I'm not used to having to use this editor. I have vs code setup in a way that prevents things like this